### PR TITLE
AODN QLD WAVE NRT - add script to retrieve ids to download from harve…

### PR DESCRIPTION
**Pull request by lbesnard**
…ster

@bpasquer  I did a few modification to the cron logic from the original script you sent me.

### CODE REVIEW

**Comment by lbesnard** [comment]
NetCDF's on
https://cloudstor.aarnet.edu.au/plus/apps/files/?dir=/WAVE&fileid=2265665290

File: AODN/AODN-WAVE-DM/DES-QLD-WAVE-DM/lib/qld_wave_parameter_mapping.csv [hunk 1]
```diff
@@ -0,0 +1,11 @@
+ORIGINAL_VARNAME,VARNAME,STANDARD_NAME,LONG_NAME,UNITS,COMMENT,VALID_MIN,VALID_MAX,REFERENCE_DATUM,POSITIVE,ANCILLARY_VARIABLES
+Tz,WPMH,sea_surface_wave_zero_upcrossing_period,,s,The average of the zero up-crossing wave periods (in seconds) in a wave record.,0,50,,,
+Hs,WHTH,sea_surface_wave_significant_height,Significant height of waves on the water body,m,an average of the highest third of the waves in a record (26.6 minute recording period).,0,50,100,,
```
[the remark this answers is not in the document]
Comment (lbesnard) [reply AODN/AODN-WAVE-DM/DES-QLD-WAVE-DM/lib/qld_wave_parameter_mapping.csv]: oh of course ! you re doing it ?

[the remark this answers is not in the document]
Comment (bpasquer) [reply AODN/AODN-WAVE-DM/DES-QLD-WAVE-DM/lib/qld_wave_parameter_mapping.csv]: yes

**Comment by bpasquer** [comment]
I've added all my edits. Happy to merge if you are ready

**Comment by lbesnard** [comment]
I'm happy.

**Pull request merged**